src/coreclr/tools/aot/DependencyGraphViewer/app.manifest

```diff
@@ -1,25 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
   <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
-  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
```

Comment:
I'm not sure if it's this part or elsewhere but I really liked the fact that VisualStudio could restart itself in administrator mode if I wanted to use ETW events. Now you get a message that tells you to do it but leaves you on your own to figure out how and it's extra work that the tool can do for you. Maybe we could have something like:
To turn on ETW events you need to be Administrator, would you like to restart the program as an admin process?

Reply:
I think we should include this in a different set of UI updates to the DependencyGraphViewer (including removing some of the pages when viewing nodes)